interfaces/apparmor/template.go

```diff
@@ -233,13 +233,15 @@ var defaultTemplate = []byte(`
 
   # Writable home area for this version.
   owner @{HOME}/snap/@{SNAP_NAME}/@{SNAP_REVISION}/** wl,
+  owner @{HOME}/snap/@{SNAP_NAME}/shared/** wl,
```

Comment:
"common" doesn't really have any access-related concurrency connotation that I'm aware of, and searching for these terms seems to agree with that. Do you have any reference pointing out otherwise?

The fact multiple entities can access it is intended, though. That's exactly what "not versioned" means, and it's always more pleasing to label something after what it means than after what it does not mean.